### DISCUSSION TITLE
How do I check index? If it exist?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18198] [Q&A and General discussion] [answered]
Is there a way to check index? 
How do I get information about collection (segment size and so on..)

I am not sure if Flat idex is present... 
In GUI zilliz/attu:latest there is a button to create index.

In menu I can't select Flat index... the create button simple is not active. (Only for Flat index) for al other indexes it is available.

I created my shema using following code

```
fields = [
    FieldSchema(name="pk", dtype=DataType.INT64, is_primary=True, auto_id=False),   
    FieldSchema(name="embeddings", dtype=DataType.FLOAT_VECTOR, dim=512)
]
schema = CollectionSchema(fields, "photos schema")
hello_milvus = Collection("photos", schema)

index = {
    "index_type": "FLAT",
    "metric_type": "L2",
    "params":{}
}
hello_milvus.create_index("embeddings", index)

```

**Comment by yhmo** [comment, marked as answer]
Print index information:

`print([index.params for index in collection.indexes])`

**Reply by martinenkoEduard** [reply]
It returns an empty list for supposed Flat index...

**Reply by yhmo** [reply]
"Flat" actually is "no index".
